src/gallia/commands/scan/uds/services.py

```diff
@@ -80,10 +80,10 @@ def configure_parser(self) -> None:
     async def main(self, args: Namespace) -> None:
         self.result: list[tuple[int, int]] = []
         self.ecu.max_retry = 1
-        found: dict[int, dict[int, Any]] = {}
+        self.found: dict[int, dict[int, Any]] = {}
```

Comment:
A unified interface for scanner results needs to be defined; this is not decided yet. So for these first tests, let's use your solution. We could adjust this later.